ci/common/common.sh

```diff
@@ -104,28 +119,31 @@ commit_subtree() {(
 
   git add --all .
 
-  [[ ${CI} == true ]] && {
+  if is_ci_build --silent ; then
     # Commit on Travis CI.
-    if [[ -z "${GH_TOKEN}" ]]; then
-      echo "GH_TOKEN not set, not committing."
-      echo "To test pull requests, see instructions in README.md."
-      return 1
-    fi
-
     git config --local user.name ${GIT_NAME}
     git config --local user.email ${GIT_EMAIL}
 
     git commit -m "${CI_TARGET//-/ }: Automatic update." || true
+
+    if test -z "${GH_TOKEN}" ; then
```

Comment:
`set -u` is not enabled in other scripts. After a discussion with other shell script author I started to think he has a point and it should be enabled in scripts, just like `set -e` is enabled in our scripts now. And other scripts were modified to be all-green when running from a PR, except that they should not post anything (with the exception of sync-lp-mirrors which does not *do* anything).